ddtrace/tracer/tracer.go

```diff
@@ -151,6 +153,20 @@ func newUnstartedTracer(opts ...StartOption) *tracer {
 	} else {
 		writer = newAgentTraceWriter(c, sampler)
 	}
+	appTags := map[string]interface{}{
+		"_dd.poc.go.goroot":       runtime.GOROOT(),
+		"_dd.poc.runtime.version": runtime.Version(),
+	}
+	repoURL, repoBuildPath, commitSha := rootinternal.GitRepositoryURL(), rootinternal.GitRepositoryBuildPath(), rootinternal.GitCommitSha()
+	if len(repoURL) > 0 {
```

Comment:
```suggestion
	if v := rootinternal.GitRepositoryURL(); v != "" {
```
I think it's more common sense.